pybossa/view/projects.py

```diff
@@ -3778,6 +3778,8 @@ def integrate_ext_config(config_dict):
     reset_presented_time = project.info.get("reset_presented_time") or False
     response = dict(template='/projects/summary.html',
                     external_config_dict=json.dumps(ext_config_dict),
+                    authorized_services=ext_config.get('authorized_services', {}).get(current_app.config.get('AUTHORIZED_SERVICES_KEY', {}), []),
```

Comment:
the nested `.get` should fall back to a hashable type, like `""`. So something like
```
authorized_services=ext_config.get('authorized_services', {}).get(current_app.config.get('AUTHORIZED_SERVICES_KEY',''), []),
```